.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,12 @@
+# This workflow runs e2e tests as a Github Action for every pull request created against this repo
+name: e2e-tests
+on:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *' # daily to pick up releases
+jobs:
+  run-e2e-kind:
```

Comment:
Nit, but my preference is not to put "kind" in the job name. The fact that this makefile target happens to be using kind clusters is an implementation detail.